activiti-spring-process-extensions/src/main/java/org/activiti/spring/process/ExtensionVariableTypes.java

```diff
@@ -0,0 +1,38 @@
+package org.activiti.spring.process;
+
+import java.text.Format;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public enum ExtensionVariableTypes {
+
+    BOOLEAN("boolean",Boolean.class),
```

Comment:
@balsarori @ryandawsonuk having an ENUM with hardcoded types is quite limited and it will cause problems. What I would suggest is to think about other options to make sure that we can register new types.

Reply:
@salaboy you're thinking we could use the [variableTypes as the engine does](https://github.com/Activiti/Activiti/blob/8b90d1095c221880a43f16cdcf5f8e145dff2267/activiti-engine/src/main/java/org/activiti/engine/impl/cfg/ProcessEngineConfigurationImpl.java#L1954) so that when additional types are added by projects extending the engine then they would also be usable ?